### ISSUE TITLE
Softmax in get_segmentation

### ISSUE THREAD

**Issue by mavaylon1** [divamgupta/image-segmentation-keras#275]
Why is there a softmax in get_segmentation when the loss is categorical cross entropy? Categorical Cross Entropy is a Softmax activation plus a Cross-Entropy loss, so having a softmax at the end of the model is doing softmax twice.

**Comment by divamgupta** [comment]
In Keras, Categorical Cross Entropy is NOT  Softmax activation plus a Cross-Entropy loss
